### DISCUSSION TITLE
Add VTX Administrator to TX Modules that have joystick/LCD screens

### DISCUSSION THREAD

**Discussion by chaserporter** [ExpressLRS/ExpressLRS#1502] [Ideas]
I have the AxisFlying THOR TX module and I'm using it with a Spektrum DX9. I would really like to be able to use the VTX Administrator feature with the LCD screen and joystick since Spektrum doesn't support Lua scripts.  I'm currently still using dsmx telemetry receivers in my race fleet because I can change my band/channel from the radio screen. I would happily convert the fleet to elrs if this feature could be implemented.

Very excited for all of the rapid development and constant new features. Thanks for all the hard work!

**Comment by pkendall64** [comment]
Theres a PR open for that! It'll be coming in a future major version.

#1481

**Reply by chaserporter** [reply]
Awesome! Thanks!
